README.md

```diff
@@ -95,55 +95,59 @@ fn main() {
         .run();
 }
 
+/// set up a simple 3D scene
 fn setup(
     mut commands: Commands,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
-    // Plane
+    // circular base
     commands.spawn((
         RigidBody::Static,
-        Collider::cuboid(8.0, 0.002, 8.0),
+        Collider::compound(vec![(
+            Position(Vec3::ZERO),
+            Rotation(Quat::from_rotation_x(-std::f32::consts::FRAC_PI_2)),
+            Collider::cylinder(0.002, 4.0),
+        )]),
```

Comment:
This is meant to be an extremely basic example that demonstrates the basic API, so this shouldn't be using a compound collider and `Position` and `Rotation` here. It's needed since you rotate the circle mesh, which also rotates this cylinder, but it makes the example way too confusing for an introductory example.

I'd just use the old cuboid/plane here, for both the mesh and collider, since it's a lot simpler. As mentioned above this example, this is a modified version of Bevy's example, so we can change some things. We could also just remove the link to the example completely and have this be a standalone thing